frontend/BoardGUI.py

```diff
@@ -33,26 +35,45 @@ def draw_piece(self, piece, position):
         else:
             self.game.screen.blit(self.white_piece_image, position)
 
+    # def get_cell_clicked(self):
```

Comment:
Lets remove the commented code. We can access it later through the version history if we want to add it back.